Classes/CRON/FormBuilder/Controller/FormBuilderController.php

```diff
@@ -88,6 +93,27 @@ public function submitPendingAction()
         $this->view->assign('responseElements', $this->request->getInternalArgument('__responseElements'));
     }
 
+    protected function isBot($time) {
+        $minTime = 5;
+        $maxTime = 3600;
+        $encryptedTime = $time;
+        $key = 'honypot876574';
+        $cipher = 'aes-128-cbc';
+        $iv = '0259847523614897';
+        $decryptedTime = openssl_decrypt($encryptedTime, $cipher, $key, 0, $iv);
+
+        if($decryptedTime) {
+
+            $currentTime = new DateTime();
+            $currentTime = $currentTime->getTimestamp();
+            if( ($currentTime - $decryptedTime) <= $minTime || ($currentTime - $decryptedTime) >= $maxTime ) {
```

Comment:
@AndreEtienne Hast du getestet, ob das funktioniert?
Laut Doku gibt DateTime::getTimeStamp die Unix-Time zurück, also in Millisekunden. 3600 Millisekunden sind schnell vorbei.
Außerdem gibt openssl_decrypt laut Doku einen String zurück und du verwendest darauf die mathematischen Operatoren. Das sollte man besser casten und auch abfangen, ob der Benutzer nicht in dem Feld irgendeinen Quatsch mitschickt, der gar nicht zu einer Zahl decryptet wird.

Reply:
Jop das funktioniert wie erwartet auch mit der Zeit. Habe deinen Rat angeommen und das ganze noch als Int geparset.